apps/docs/docs/api.md

```diff
@@ -78,19 +78,19 @@ This method decorator allows for creating questions to ask the user. For informa
-| transform | `Function` | false | A transform method that takes the user's input and transforms it before printing it back to the terminal. This does not have an impact on the incoming value itself. |
-| when | `Function` or `boolean` | false | A function or boolean to determine if a question should be asked or not |
-| pageSize | `number` | false | The number of lines to render for `list`, `rawlist`, `checkbox`, and `expand` question |
-| prefix | `string` | false | Changes the _prefix_ message |
-| suffix | `string` | false | Changes the _suffix_ message |
-| askAnswered | `boolean` | false | Force the prompt even if the question has been answered. Defaults to `false` |
-| loop | `boolean` | Enable loop listing. Defaults to `true` |
+| type | `"input"`, `"number"`, `"confirm"`, `"list"`, `"rawlist"`, `"expand"`, `"checkbox"`, `"password"`, or `"editor"` | false | The kind of question for inquirer to ask. Each question has slightly different options, so pay attention to the type. Defaults to `"input"`. |
+| name | `string` | true | The name of the question's input. This is used when being passed back from the `InquirerService` and can be used with the `@*For()` decorator factories described below. |
+| message | `string` | true\* | The question to ask. This will be overwritten by the [`@DefaultFor()`](#defaultfor) counterpart. |
+| default | `string`, `number`, `boolean`, `array` | false | The default value for the question's input. This will be overwritten by the [`@ChoicesFor()`](#choicesfor) counterpart. |
+| choices | `array` | true | The choices to be defined for a `list`, `rawlist`, `checkbox`, and `expand` question. The array can be of strings, or numbers, or objects containing a `name`, `value`, and `short` property. This will be overwritten by the [`@ChoicesFor()`](#choicesfor) counterpart. |
+| validate | `Function` | false | A validation function that ensures the input is correct. On failed validation the question is re-asked. This will be overwritten by the [`@ValidateFor()`](#validatefor) counterpart. |
+| transform | `Function` | false | A transform method that takes the user's input and transforms it before printing it back to the terminal. This does not have an impact on the incoming value itself. This will be overwritten by the [`@TransformFor()`](#transformfor) counterpart. |
+| when | `Function` or `boolean` | false | A function or boolean to determine if a question should be asked or not. This will be overwritten by the [`@WhenFor()`](#whenfor) counterpart. |
+| pageSize | `number` | false | The number of lines to render for `list`, `rawlist`, `checkbox`, and `expand` question. |
+| prefix | `string` | false | Changes the _prefix_ message. |
+| suffix | `string` | false | Changes the _suffix_ message. |
+| askAnswered | `boolean` | false | Force the prompt even if the question has been answered. Defaults to `false`. |
+| loop | `boolean` | false | Enable loop listing. Defaults to `true`. |
```

Comment:
Do you think instead of adding it to each of these, we could do something like a `hint` or `info` with wording like

```
You can use the `@*For()` decorator factories to override the options in the `@Question()` decorator options.
```

Reply:
sounds better. Right after this table?

Reply:
Yeah immediately after it

Reply:
![image](https://user-images.githubusercontent.com/13461315/147520000-eac5aad2-922a-4158-a6a8-1984f29fbfd7.png)

Reply:
I choose _note_ to indicate internal stuff

Reply:
![image](https://user-images.githubusercontent.com/13461315/147520658-d2fd787e-5dc9-48b1-af24-0ca3af575a95.png)